website/docs/docs/deploy/continuous-integration.md

```diff
@@ -16,7 +16,7 @@ Using CI helps:
 
 ## How CI works
 
-When you [set up CI jobs](/docs/deploy/ci-jobs#set-up-ci-jobs), dbt Cloud listens for webhooks from your Git provider indicating that a new PR has been opened or updated with new commits. When dbt Cloud receives one of these webhooks, it enqueues a new run of the CI job. If you want CI checks to run on each new commit, you need to mark your PR as **Ready for review** in your Git provider &mdash; draft PRs _don't_ trigger CI jobs. 
+When you [set up CI jobs](/docs/deploy/ci-jobs#set-up-ci-jobs), dbt Cloud listens for webhooks from your Git provider indicating that a new PR has been opened or updated with new commits. When dbt Cloud receives one of these webhooks, it enqueues a new run of the CI job. 
```

Comment:
do you think it's worth mentioning that CI only runs on draft PRs if configured in the job settings?

Reply:
i don't think so. we added this here because it was a potential pitfall. and we didn't want this info buried in a faq.